storybook/components/component-name/component-name-decorator.jsx

```diff
@@ -0,0 +1,17 @@
+import cx from "classnames";
+import { ComponentName } from "vibe-storybook-components";
+import "./component-name-decorator.scss";
```

Comment:
prefer chaning to CSS Modules. if there aren't plenty let's just get rid of the global css

Reply:
Decided to do it later